src/lib/pages/backoffice/Acquisition/Order/OrderDetails/OrderLines.js

```diff
@@ -100,6 +103,13 @@ OrderLineRightColumn.propTypes = {
 };
 
 export const OrderLine = ({ line, LeftColumn, MiddleColumn, RightColumn }) => {
+  const renderSubtitle = (line) => {
```

Comment:
same as above. Maybe it should be a small utility function instead of having the code repeated multiple times?